Makefile

```diff
@@ -311,13 +336,17 @@ fawiki_tuning_reports: \
 #	./utility extract_labelings \
 #		/mnt/data/xmldatadumps/public/frwiki/20150602/frwiki-20150602-pages-meta-history*.xml*.bz2 > $@
 
-datasets/frwiki.labelings.20151202.json:
+#datasets/frwiki.labelings.20151202.json:
+#	./utility extract_labelings \
+#	  /mnt/data/xmldatadumps/public/frwiki/20151202/frwiki-20151202-pages-meta-history*.xml*.bz2 > $@
+
+datasets/frwiki.labelings.20181201.json:
 	./utility extract_labelings \
-	  /mnt/data/xmldatadumps/public/frwiki/20151202/frwiki-20151202-pages-meta-history*.xml*.bz2 > $@
+	  /mnt/data/xmldatadumps/public/frwiki/20181201/frwiki-20181201-pages-meta-history*.xml*.bz2 > $@
```

Comment:
Not the fault of this patch: I don't like the absolute path, when we templatize this should come from configuration.

Reply:
Yeah.  Now that we have more standardized dates for the XML dumps, this is a good idea.